packages/assets-controllers/src/Standards/NftStandards/ERC1155/ERC1155Standard.ts

```diff
@@ -26,41 +26,39 @@ export class ERC1155Standard {
    * @param address - ERC1155 asset contract address.
    * @returns Promise resolving to whether the contract implements ERC1155 URI Metadata interface.
    */
-  contractSupportsURIMetadataInterface = async (
```

Comment:
Just curious was there any functional purpose to changing these from arrow functions?

Reply:
They were changed to make them mockable with jest.  Don't expect any functional changes.